packages/common/src/controllers/AppController.ts

```diff
@@ -81,9 +82,10 @@ export default class AppController {
     // update settings in the config store
     useConfigStore.setState({ settings });
 
-    // when an integration is set, we initialize the AccountController
+    // when an integration is set, we initialize AccountController and AccessController
     if (integrationType) {
       await getModule(AccountController).initialize(url, refreshEntitlements);
+      await getModule(AccessController).initialize();
```

Comment:
Accounting for if access controller is not used:
```suggestion
      await getModule(AccessController)?.initialize();
```